dbt_subprojects/solana/models/_sector/dex/orca_whirlpool/orca_whirlpool_base_trades_v2.sql

```diff
@@ -0,0 +1,251 @@
+ {{
+  config(
+
+        schema = 'orca_whirlpool',
+        alias = 'base_trades_v2',
```

Comment:
How about we rename it to `orca_whirlpool_v2.base_trades`?
```suggestion
        schema = 'orca_whirlpool_v2',
        alias = 'base_trades',
```